src/web3/saga.ts

```diff
@@ -308,8 +308,9 @@ export function* createAndAssignCapsuleAccount() {
     let account = ''
     try {
       account = yield call([wallet, wallet.addAccount])
-      const privateKeyShare: string = wallet.getKeyshare(account)
-      yield call(storeCapsuleKeyShare, privateKeyShare, account)
+      void wallet.getKeyshare(account).then((privateKeyShare) => {
```

Comment:
getKeyshare became `async,` and we do not support async generators. I guess this persisting can be completely wiped-out.

Reply:
are the implications of this only that we don't store the keyShare in memory anymore here?

Reply:
Actually, there's no implication. I just made the old logic still functioning. In the old logic, we had a mechanism to store the private key, but after that, it was never loaded back. It was not possible to sign any transaction after restoring the wallet (re-launching the app).
Still, we do not rely on the old mechanism to store; we incorporated it directly in the Wallet class. However, I left it here to minimize my changes.